angularjs-1.5-basics/src/app/angular-1.5-basics-main.mock.js

```diff
@@ -0,0 +1,10 @@
+/*eslint-env jasmine */
+/*global module:false, inject:false */
+'use strict';
+
+angular.module('angular-1.5-basics-main.mocks', ['angular-1.5-basics-main','ngMockE2E']).config(function () {
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->